docs/system-settings/key-profiles.md

```diff
@@ -66,30 +66,33 @@ the fader to 0.
 ### Groups
 The blue, grey or touch key can be allocated to **Disabled**,
 **Select Group**, **Flash Fixtures**, **Timed Flash**, **Flash Master**, **Timed Flash
-Master**, **Swop Fixtures**.
+Master**, **Swop Fixtures** - see [Fixture Groups](../controlling-fixtures/fixture-groups)
+for more details.
 
-You can select if the group function takes
-precedence over the flash function (if a handle has both group and flash
-functions) or vice versa.
+>   The option **Group/Flash takes precedence** only applies to the Pearl Expert
+    where it is possible to program a playback on a fader handle and use its
+    flash button as group selector.
 
 ### Palettes
 
-> Only effective on Pearl Expert.
+The palette key or touch button can be allocated to **Disabled** or **Select Palette**.
 
-The palette (grey) key can
-be allocated to **Disabled** or **Select Palette**. You can select if the
-palette function takes precedence over the flash function (if a handle
-has both palette and flash functions) or vice versa.
+With **Palette is fired ignoring/with its times** you select whether to take palette
+times into account when selecting a palette, see [Timed Palettes](../palettes/timing-with-palettes).
+This setting is particularly useful when [Busking with palettes](../playback-controls#busking-with-palettes).
```

Comment:
These both need to link to the .md files

Reply:
just committed

Reply:
The test failed with the error:

````
Could not find "/home/runner/work/TitanManual/TitanManual/docs/playback-controls.md"
Linked from "/home/runner/work/TitanManual/TitanManual/docs/system-settings/key-profiles.md" (with the link text "Busking with palettes")
````